example/cds-hooks-api-guide/src/suggestions-links-fhir/suggestions-links-fhir.js

```diff
@@ -46,205 +50,89 @@ const handler = async (request) => {
     smokingStatus
   );
   const riskThresholdString = riskThreshold(riskScore[0]);
-  // defining the cards
-  let cards = [];
-  if (riskScore[1] == "info") {
-    // push just the link card if the patients score is on the lower side
-    cards.push(
-      new Card({
-        detail: `More information on this score:`,
-        source: {
-          label: "Reynold's Risk Score",
-          url: "https://pubmed.ncbi.nlm.nih.gov/17299196/",
-        },
-        summary: `Reynolds risk score: ${riskScore[0]}`,
-        indicator: riskScore[1],
-        links: [
-          {
-            label: "Launch cardiac health SMART app",
-            url: "https://divine-meadow-3697.fly.dev/launch.html",
-            type: "smart",
-          },
-        ],
-      })
-    );
-  } else if (riskScore[1] == "warning") {
-    // push the link card
-    cards.push(
-      new Card({
-        detail: `More information on this score:`,
-        source: {
-          label: "Reynold's Risk Score",
-          url: "https://pubmed.ncbi.nlm.nih.gov/17299196/",
+  /**
+   * Defining the cards.
+   * This assumes that the reynolds risk score card will be sent every time
+   */
+  let cards = [
+    new Card({
+      detail: `More information on this score:`,
+      source: {
+        label: "Reynold's Risk Score",
+        url: "https://pubmed.ncbi.nlm.nih.gov/17299196/",
+      },
+      summary: `Reynolds risk score: ${riskScore[0]}`,
+      indicator: riskScore[1],
+      links: [
+        {
+          label: "Launch cardiac health SMART app",
+          url: "https://divine-meadow-3697.fly.dev/launch.html",
```

Comment:
I redeployed this to https://smart-cardiac-risk.fly.dev - can you check to make sure it works and use that URL instead

Reply:
Also opened PR on that project to just merge the config for that

Reply:
This is a necessary one

Reply:
Confirmed working